test/expected/resolve_connection_filter.out

```diff
@@ -230,6 +231,181 @@ begin;
  {"data": null, "errors": [{"message": "Error transpiling Is filter value type"}]}
 (1 row)
 
+    rollback to savepoint a;
+    -- cs - array column contains the input scalar
+    select jsonb_pretty(
+        graphql.resolve($$
+            {
+              accountCollection(filter: {tags: {cs: "customer"}}) {
+                edges {
+                  node {
+                    id
+                  }
+                }
+              }
+            }
+        $$)
+    );
+          jsonb_pretty           
+---------------------------------
+ {                              +
+     "data": {                  +
+         "accountCollection": { +
+             "edges": [         +
+                 {              +
+                     "node": {  +
+                         "id": 1+
+                     }          +
+                 },             +
+                 {              +
+                     "node": {  +
+                         "id": 2+
+                     }          +
+                 }              +
+             ]                  +
+         }                      +
+     }                          +
+ }
+(1 row)
+
+    rollback to savepoint a;
+    -- cd - array column is contained by input scalar (aka, the only value in the array column is the input scalar)
```

Comment:
New test case: array column is contained by input scalar (aka, the only value in the array column is the input scalar)